### DISCUSSION TITLE
Welcome to snuggsi Discussions!

### DISCUSSION THREAD

**Discussion by snuggs** [devpunks/snuggsi#214] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brandondees** [comment]
👋🏻 I'm Brandon Dees, a Senior Software Engineer at a medtech startup. I've been a web developer since 2005, Rails dev since about 2008, professional since 2012. I met @snuggs in the https://www.rubyonrails.link/ ruby community slack years ago and am pretty interested in the snuggsi approach to webcomponents as a more viable alternative to things like Polymer.

**Comment by miketout** [comment]
Hi, I'm interested in using snuggsi, and I'm hoping to connect with one of the devs/maintainers, especially if you have interest in consulting/helping out with adapting it to a very interesting project. Please let me know. Thanks!
